### ISSUE TITLE
$global variables not working on Android

### ISSUE THREAD

**Issue by tcguy** [jasonelle-archive/jasonelle-v2#40]
I can't seem to get $global variables to work after closing down the app and relaunching it on Android devices.  $cache works fine.

```
{
    "$jason": {
        "head": {
            "title": "Global Variable Test",
            "actions": {
                "$pull": {
                    "type": "$flush",
                    "success": {
                        "type": "$reload"
                    }
                },
                "store": {
                    "type": "$global.set",
                    "options": {
                        "variable": "{{$get.variable || ''}}"
                    },
                    "success": {
                        "type": "$util.banner",
                        "options": {
                            "title": "Current global variable",
                            "description": "{{$global.variable}}"
                        }
                    }
                },
                "display": {
                    "type": "$util.alert",
                    "options": {
                        "title": "Current global variable",
                        "description": "{{$global.variable}}"
                    }
                }
            }
        },
        "body": {
            "sections": [{
                "items": [{
                    "type": "vertical",
                    "components": [{
                            "type": "label",
                            "text": "{{$global.variable}}"
                        },
                        {
                            "type": "space",
                            "style": {
                                "height": "20"
                            }
                        },
                        {
                            "type": "textfield",
                            "name": "variable",
                            "placeholder": "Variable to set",
                            "keyboard": "text",
                            "style": {
                                "background": "#FFFFFF",
                                "width": "250",
                                "size": "16",
                                "corner_radius": "4"
                            }
                        },
                        {
                            "type": "space",
                            "style": {
                                "height": "20"
                            }
                        },
                        {
                            "type": "horizontal",
                            "components": [{
                                    "type": "button",
                                    "text": "Store",
                                    "style": {
                                        "width": "100",
                                        "height": "50",
                                        "background": "green",
                                        "color": "#ffffff",
                                        "size": "22",
                                        "corner_radius": "4"
                                    },
                                    "action": {
                                        "trigger": "store"
                                    }
                                },
                                {
                                    "type": "space",
                                    "style": {
                                        "width": "20"
                                    }
                                },
                                {
                                    "type": "button",
                                    "text": "Display",
                                    "style": {
                                        "width": "100",
                                        "height": "50",
                                        "background": "green",
                                        "color": "#ffffff",
                                        "size": "22",
                                        "corner_radius": "4"
                                    },
                                    "action": {
                                        "trigger": "display"
                                    }
                                }
                            ]
                        }
                    ],
                    "style": {
                        "align": "center"
                    }
                }]
            }],
            "style": {
                "background": "gray"
            }
        }
    }
}
```

**Comment by clsource** [comment]
Maybe this is related on how https://developer.android.com/guide/topics/data/data-storage.html#pref are used in the global action.

https://github.com/jasonelle/jasonette-android/blob/master/app/src/main/java/com/jasonette/seed/Action/JasonGlobalAction.java

and the launcher code 

https://github.com/jasonelle/jasonette-android/blob/master/app/src/main/java/com/jasonette/seed/Launcher/Launcher.java#L93

**Comment by panterozo** [comment]
Hi @tcguy... Fixed it in commit #d3b511a, now in develop branch. Soon will be merge into master. You should retry
